guides/office/kitchen.md

```diff
@@ -9,9 +9,31 @@ We love our coffee at Made Tech. We have fresh beans regularly delivered to the
  - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
  - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
 
-## Sink regime
+## Dirty glasses, cutlery, and crockery regime
 
-We all appreciate a clear and useable sink. Please ensure:
+We all appreciate a clean kitchen, with no dirty tableware, and a clear sink. This is why everyone should put their dirty tableware in the dishwasher.
```

Comment:
There's a lot of tableware here.

How about "We all appreciate a clean and orderly kitchen and a clear sink..."

Reply:
Agreed, this one got out of hand, will fix!